homeassistant/components/zwave_js/discovery.py

```diff
@@ -666,6 +666,17 @@ def get_config_parameter_discovery_schema(
         ),
         required_values=[SIREN_TONE_SCHEMA],
     ),
+    # select
+    # protection CC
+    ZWaveDiscoverySchema(
+        platform="select",
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.PROTECTION},
+            property={"local", "rf"},
+            type={"number"},
+        ),
+        entity_registry_enabled_default=False,
```

Comment:
Why disabled by default? It's not like Basic Alarm V1 values where there is a (usually) better alternative.

Reply:
I made the assumption that they were not commonly used. This is easy enough to switch, but I don't use the protection CC myself so can't weigh in

Reply:
I think it would be worth enabling by default. Most people probably don't know it exists, but its handy (especially for folks with small kids that like to press buttons 🙂)